buildenv/jenkins/getDependency

```diff
@@ -19,7 +19,7 @@ def testBuild() {
 			if( params.BUILD_TYPE == "systemtest" ){
 				sh 'curl -OLJks "https://api.adoptopenjdk.net/v3/binary/latest/8/ga/linux/x64/jdk/hotspot/normal/adoptopenjdk"'
 				sh 'mkdir ${WORKSPACE}/j2sdk-image'
-				sh 'tar -xzf OpenJDK8U-jdk_x64_linux_hotspot*.gz -C ${WORKSPACE}/j2sdk-image --strip-components 1'
+				sh -c 'mkdir dir.$$ && cd dir.$$ && gzip -cd ../OpenJDK8U-jdk_x64_linux_hotspot*.gz | tar xpf - && mv */* ${WORKSPACE}/j2sdk-image && cd .. && rm -rf dir.$$'
```

Comment:
This script only runs on xlinux. I do not think this needs to be updated.
https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/buildenv/jenkins/getDependency#L3